### PR TITLE
Remove asset RPCs

### PR DESCRIPTION
**Pull request by majecty**
Foundry has a module system to make a user be able to create its logic using it. If the user needs a custom asset, they will implement it using the module. So Foundry does not need the Asset system.

I'll create a few more PRs to remove all the Asset code.

~This PR depends on #26~

### CODE REVIEW

File: rpc/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-2019 Kodebox, Inc.
+// Copyright 2018 Kodebox, Inc.
```
Comment (sgkim126) [inline rpc/src/lib.rs]: Would you check it?

**Comment by sgkim126** [comment]
IMO we should remove some e2e tests also. Are there tests that use the removed RPCs?

File: rpc/src/lib.rs (hunk 1 above)
Comment (majecty) [reply rpc/src/lib.rs]: Sure. I fixed it.

File: rpc/src/v1/impls/chain.rs [hunk 2]
```diff
@@ -14,36 +14,33 @@
...
-    AccountData, AssetClient, BlockId, EngineInfo, ExecuteClient, MiningBlockChainClient, Shard, TermInfo, TextClient,
-};
-use ccrypto::Blake;
+use ccore::{AccountData, BlockId, EngineInfo, ExecuteClient, MiningBlockChainClient, Shard, TermInfo, TextClient};
 use cjson::scheme::Params;
 use cjson::uint::Uint;
 use ckey::{public_to_address, NetworkId, PlatformAddress, Public};
 use cstate::FindActionHandler;
-use ctypes::transaction::{Action, ShardTransaction as ShardTransactionType};
+use ctypes::transaction::{Action};
```
Comment (sgkim126) [inline rpc/src/v1/impls/chain.rs]:
```rust
use ctypes::transaction::Action;
```

File: rpc/src/v1/impls/chain.rs [hunk 3]
```diff
@@ -14,36 +14,33 @@
...
 use ctypes::{BlockHash, BlockNumber, ShardId, Tracker, TxHash};
-use primitives::{Bytes as BytesArray, H160, H256};
+use primitives::{H256};
```
Comment (sgkim126) [inline rpc/src/v1/impls/chain.rs]:
```rust
use primitives::H256;
```

File: rpc/src/v1/impls/chain.rs [hunk 4]
```diff
@@ -14,36 +14,33 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use std::convert::{TryFrom, TryInto};
+use std::convert::{TryFrom};
```
Comment (sgkim126) [inline rpc/src/v1/impls/chain.rs]:
```rust
use std::convert::TryFrom;
```

File: rpc/src/v1/impls/chain.rs (hunk 2 above)
Comment (majecty) [reply rpc/src/v1/impls/chain.rs]: Thanks. I'll fix them.

**Comment by majecty** [comment]
I fixed a merge conflict

**Pull request merged**